### ISSUE TITLE
Error when running tests on React Native project

### ISSUE THREAD

**Issue by dgrcode** [antoniopresto/react-native-local-mongodb#23]
I'm getting this error when trying to run tests on a react native project.

```
Couldn't find preset "env" relative to directory "/Users/daniel/Projects/Native/myProject/node_modules/react-native-local-mongodb"

at node_modules/babel-core/lib/transformation/file/options/option-manager.js:293:19
      at Array.map (native)
      at OptionManager.resolvePresets (node_modules/babel-core/lib/transformation/file/options/option-manager.js:275:20)
      at OptionManager.mergePresets (node_modules/babel-core/lib/transformation/file/options/option-manager.js:264:10)
      at OptionManager.mergeOptions (node_modules/babel-core/lib/transformation/file/options/option-manager.js:249:14)
      at OptionManager.init (node_modules/babel-core/lib/transformation/file/options/option-manager.js:368:12)
      at File.initOptions (node_modules/babel-core/lib/transformation/file/index.js:212:65)
      at new File (node_modules/babel-core/lib/transformation/file/index.js:135:24)
      at Pipeline.transform (node_modules/babel-core/lib/transformation/pipeline.js:46:16)
```

My project was created with `create-react-native-app`, and I'm running the tests using 'node node_modules/jest/bin/jest.js --watch'.

My best guess is that there's something not set in this repo, but this is the first time I see this issue, so I'm not sure about the root cause.

**Comment by holasebas** [comment]
same here  😅

**Comment by holasebas** [comment]
rm -rf .babelrc on the project root then running ./node_modules/react-native/scripts/packager.sh --reset-cache solved that problem for me. No idea why.

**Comment by dgrcode** [comment]
I run `./node_modules/react-native/scripts/packager.sh --reset-cache` but it didn't solve my problem.

I don't really want to remove `.babelrc`

**Comment by antoniopresto** [comment]
Are you running tests in the node_modules folder too? Can you ignore it?

**Comment by antoniopresto** [comment]
try this config
```
"jest": {
    "preset": "jest-expo",
    "collectCoverageFrom": [
      "**/*.{js,jsx}",
      "!**/node_modules/**",
      "!**/vendor/**"
    ],
    "transformIgnorePatterns": [
      "[/\\\\]node_modules[/\\\\]react-native-local-mongodb"
    ]
  }

**Comment by antoniopresto** [comment]
fixed in 1.9.2

**Comment by dgrcode** [comment]
I bumped my version to 1.9.2 and it works like a charm.

Thanks!
